src/test/java/showcase/neodymium/tests/pdf/PdfTextContentCheckTest.java

```diff
@@ -23,11 +23,6 @@
  * implemented in Java and the execution environment is configured with Maven any available 3rd party technology can be
  * added to the test automation.<br>
  * This show case gives an basic example how to implement an PDF content check using pdfbox.<br>
- * <br>
```

Comment:
Please add this as information to documentation, that the dependency is required but already prepared by us.